### DISCUSSION TITLE
[@types/inputmask] doesn't contain latest inputmask definitions

### DISCUSSION THREAD

**Discussion by pks28** [DefinitelyTyped/DefinitelyTyped#58163] [Issues with a @types package]
inputmask has introduced some new options like preFillYear but these options are not yet  updated in @types/inputmask. Can somebody update these definitions because I urgently require this option in my angular application.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
@pks28 Hi. I can update definitions but I need more as you write in your comment. I need documentation, what is missing, or a link to the source code.

**Reply by pks28** [reply]
Hi @martin-badin this is the link to source code : [https://github.com/RobinHerbots/Inputmask](url) . For the datetime alias , prefillYear option which was added in the latest version is not available in typed definitions.

**Reply by filipve1994** [reply]
@martin-badin @pks28 I think you have to add the following code in the part interface Options. below Line 535

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inputmask/index.d.ts#L535

`
    /**
     * Enable/disable prefilling of the year.
     *
     * Although you can just over type the proposed value without deleting, many seems to see a problem with the year prediction.
     * This options is to disable this feature.
     *
     * @default true
     */
    prefillYear?: boolean | undefined;
`

**Reply by martin-badin** [reply]
PR is ready. Please review #58663
